can/dbc.cc

```diff
@@ -75,21 +75,23 @@ ChecksumState* get_checksum(const std::string& dbc_name) {
     s = new ChecksumState({8, -1, 7, -1, false, CHRYSLER_CHECKSUM, &chrysler_checksum});
   } else if (startswith(dbc_name, "comma_body")) {
     s = new ChecksumState({8, 4, 7, 3, false, PEDAL_CHECKSUM, &pedal_checksum});
+  } else if (startswith(dbc_name, "ford_")) {
+    s = new ChecksumState({8, 4, 0, 0, false, FORD_CHECKSUM, &ford_checksum});
   }
   return s;
 }
 
 void set_signal_type(Signal& s, ChecksumState* chk, const std::string& dbc_name, int line_num) {
   s.calc_checksum = nullptr;
   if (chk) {
-    if (s.name == "CHECKSUM") {
+    if (s.name == "CHECKSUM" || endswith(s.name, "_Cs")) {
```

Comment:
There's been discussions about this sort of thing for other manufacturers where the canonical signal names were known. The most recent I'm aware of is in #633. TL;DR: the fast thing is to rename your signals, the right thing would be creating support for tagging the existing signals in a to-be-determined manner.